tst/EnergyPlus/unit/HVACVariableRefrigerantFlow.unit.cc

```diff
@@ -714,13 +703,21 @@ namespace EnergyPlus {
 		ASSERT_EQ( DefrostWatts, VRF( VRFCond ).DefrostPower ); // defrost power calculation check
 
 		// clean up
+		VRF.deallocate();
+		VRFTU.deallocate();
+		TerminalUnitList.deallocate();
+
+		StdRhoAir = 0.0;
+		ZoneEqSizing.deallocate();
+		FinalZoneSizing.deallocate();
 		ZoneSysEnergyDemand.deallocate();
+		DesDayWeath.deallocate();
 
 	}
 
-	TEST_F( HVACFixture, VRFTest_WaterCooled ) {
-
-		static std::string const RoutineName( "VRFTest_WaterCooled" );
+	TEST_F( HVACFixture, VRFTest_GetInputFailers ) {
+		// Author: R. Raustad, FSEC
```

Comment:
This looks like it deleted a previous unit test. I'll add back in uisng a different branch.